src/style.css

```diff
@@ -1 +1,12 @@
 /* styles go here */
+.fieldsetConfig {
```

Comment:
We do not normally use camelCase here. For CSS classes there is a different naming convention.